### PR TITLE
Fix disk type’Malformed URL’ error

### PR DESCRIPTION
**Pull request by sl1pm4t**
The API expects the disk type to be a SelfLink URL, but the disk type 
name was being used (e.g. “pd-ssd”). This resulted in the following error:

```
Error applying plan:

1 error(s) occurred:

* google_compute_instance.foo: 1 error(s) occurred:

* google_compute_instance.foo: Error creating instance: googleapi: Error 400: Invalid value for field 'resource.disks[0].initializeParams.diskType': 'pd-ssd'. The URL is malformed., invalid
```

### CODE REVIEW

**Comment by sl1pm4t** [comment]
Please review @danawillow 
Thanks!

**Comment by danawillow** [comment]
Thanks @sl1pm4t! Wish we had caught this with a test instead :-/ Mind adding (or modifying) a test that sets `type`?

**Comment by sl1pm4t** [comment]
Sure thing @danawillow

File: google/resource_compute_instance_test.go [hunk 1]
```diff
@@ -1370,6 +1409,27 @@ resource "google_compute_instance" "foobar" {
...
 
+func testAccComputeInstance_bootDisk_type(instance string) string {
+	return fmt.Sprintf(`
+resource "google_compute_instance" "foobar" {
+	name         = "%s"
+	machine_type = "n1-standard-1"
+	zone         = "us-central1-a"
+
+	boot_disk {
+    initialize_params {
```
Comment (danawillow) [inline google/resource_compute_instance_test.go]: Can you fix the indentation here? It should be tabs.

**Comment by sl1pm4t** [comment]
Oops - that test doesn't work.

**Comment by danawillow** [comment]
Thanks, looks good!

**Pull request merged**